docs/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "scripts": {
-    "docs:dev": "vitepress dev",
-    "docs:build": "vitepress build",
-    "docs:preview": "vitepress preview"
+    "docs:dev": "vitepress dev docs",
+    "docs:build": "vitepress build docs",
+    "docs:preview": "vitepress preview docs"
```

Comment:
For this to work, then the `package.json` file would need to be moved to the root directory. From my tests, it works without the `docs` suffix